src/Pages/ArticleDetailPage.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+import { css } from "glamor";
+import { observer } from "mobx-react";
+
+import ReactMarkdown from "react-markdown";
+import ArticleHeader from "./../Components/ArticleHeader";
+import SmallCardPost from "./../Components/SmallCardPost";
+
+const ArticlePageObserver = observer(
+  class ArticlePageObserver extends Component {
```

Comment:
I’m curious about this double naming. I’ve never seen this approach. I usually wrap the export below in observer function.

Reply:
That should have been ArticleDetailPage. I missed it somehow.